packages/server/lib/browsers/index.js

```diff
@@ -12,15 +12,16 @@ const isBrowserFamily = check.oneOf(['chromium', 'firefox'])
 let instance = null
 
 const kill = function (unbind) {
-  // cleanup our running browser
-  // instance
+  // Clean up the browser instance that's currently running
   if (!instance) {
     return Promise.resolve()
   }
 
   return new Promise((resolve) => {
     if (unbind) {
       instance.removeAllListeners()
+
+      return resolve.apply(null, [unbind])
```

Comment:
trying to understand this change...

`unbind` is `true` only in `browsers.open`: https://github.com/cypress-io/cypress/blob/1282c70c131aab8df377a9ed92fd689aa2604946/packages/server/lib/browsers/index.js#L179-L180

the resolved value appears to be irrelevant, so you could just `resolve()`

so the new logic is that when a browser is opened and there is an `instance`, instead of killing the existing browser instance, just remove all listeners from it and resolve `kill`?

that seems like, wrong to me... `instance.kill()` never even runs with this patch, i feel like that's bound to cause issues somewhere - maybe in `cypress open`? 

and `cleanup` is never called with this patch, so `instance` will still exist